File: TextosJapones/src/main/java/org/jisho/textosJapones/model/enums/Tipo.java
package org.jisho.textosJapones.model.enums;

public enum Tipo {
	TEXTO("Texto"), MUSICA("Música"), VOCABULARIO("Vocabulário"), KANJI("Kanji");

	private String modo;

	Tipo(String modo) {
		this.modo = modo;
	}

	public String getDescricao() {
		return modo;
	}

	@Override
	public String toString() {
		return this.modo;
	}
}

File: ProcessaTexto/src/main/kotlin/br/com/fenix/processatexto/model/enums/Tipo.kt
package br.com.fenix.processatexto.model.enums

enum class Tipo(val descricao: String) {
    TEXTO("Texto"), MUSICA("Música"), VOCABULARIO("Vocabulário"), KANJI("Kanji");

    @Override
    override fun toString(): String {
        return descricao
    }
}